scripts/ci-tool.py

```diff
@@ -386,6 +426,20 @@ def main():
         fetchai_code_quality.static_analysis(
             project_root, build_root, args.fix, concurrency, args.commit, verbose=False)
 
+    if (args.build or args.lint or args.all) and sccache_path:
+        subprocess.check_call([sccache_path, '-s'])
+
+        # stop the server
+        stop_sscache_server(sccache_path)
```

Comment:
This won't be reached if the build fails at any point.

Reply:
That is correct - this feature in only intended for CI builds currently - so the what happens when if the server crashes or fails it not really an issue because everything is containerised.

If you run this locally ultimately it will also cause a compilation failure